parser-typechecker/src/Unison/Util/Pretty.hs

```diff
@@ -259,6 +259,7 @@ toPlainUnbroken p = CT.toPlain (renderUnbroken p)
 syntaxToColor :: Pretty ST.SyntaxText -> Pretty ColorText
 syntaxToColor = fmap $ annotateMaybe . fmap CT.defaultColors
 
+-- set the syntax, overriding any present syntax
```

Comment:
I added this comment while trying to figure something out, I can remove it if you like